gems/pending/disk/modules/MSCommon.rb

```diff
@@ -349,4 +363,5 @@ def self.checksum(buf, skip_offset)
     # GRRRRR - convert to actual 32-bits.
     [~csum].pack('L').unpack('L')[0]
   end
+  private_class_method :checksum
```

Comment:
Unping, I went through and git grep'd around and nothing is calling these private class methods with an explicit  receiver (not even self).  I'm good with merging this as euwe/no (e release) so we have time to test out these changes for the F-release.